packages/web/src/components/edit/fields/visibility/mergeReleaseDateValues.ts

```diff
@@ -0,0 +1,22 @@
+import moment from 'moment'
```

Comment:
i thought we were moving towards dayjs?

Reply:
yeah im quite down, just using what existed already in this case (copied from legacy field)